packages/blog2/src/components/molecules/Seo/index.tsx

```diff
@@ -10,85 +11,44 @@ const {
-    <meta name="twitter:site" content={`@${twitter}`} />
-    <meta name="twitter:title" content={title} />
-    <meta name="twitter:description" content={description} />
-    <meta name="twitter:card" content="summary_large_image" />
-    <meta property="twitter:image" content={imageUrl}></meta>
-  </>
-);
-
-type OpenGraphSeoPropsType = Pick<
-  SeoPropsType,
-  "description" | "imageUrl" | "imageHeight" | "imageWidth" | "path" | "title"
->;
-
-const OpenGraphSeo: FC<OpenGraphSeoPropsType> = ({
-  description,
-  imageUrl,
-  imageHeight,
-  imageWidth,
-  path,
-  title,
-}) => (
-  <>
-    <meta property="og:site_name" content={metaTitle} />
-    <meta property="og:title" content={title} />
-    <meta property="og:description" content={description} />
-    <meta property="og:type" content="website" />
-    <meta property="og:url" content={path} />
-    <meta property="og:image" content={imageUrl} />
-    <meta property="og:image:height" content={`${imageHeight}`} />
-    <meta property="og:image:width" content={`${imageWidth}`} />
-  </>
-);
-
 type SeoPropsType = Pick<PostType, "description" | "keywords"> & {
-  imageUrl: string;
-  imageHeight: number;
-  imageWidth: number;
+  image: ImageType;
   path: string;
   title: SanitisedString;
 };
 
 export const Seo: FC<SeoPropsType> = ({
   description,
-  imageUrl,
-  imageHeight,
-  imageWidth,
+  image,
   keywords,
   path,
   title,
 }) => (
   <NextHead>
+    {/* general */}
     <title>
       {title} | {metaTitle}
     </title>
     <meta name="description" content={description} />
     <link rel="icon" href="/profile.jpg" />
-    <meta name="image" content={imageUrl} />
+    <meta name="image" content={image.url} />
     <meta name="keywords" content={classNames(...keywords)} />
     <link rel="canonical" href={path} />
-    <OpenGraphSeo
-      description={description}
-      imageUrl={imageUrl}
-      imageHeight={imageHeight}
-      imageWidth={imageWidth}
-      path={path}
-      title={title}
-    />
-    <TwitterSeo description={description} imageUrl={imageUrl} title={title} />
+    {/* open graph */}
+    <meta property="og:site_name" content={metaTitle} />
+    <meta property="og:title" content={title} />
+    <meta property="og:description" content={description} />
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content={path} />
+    <meta property="og:image" content={image.url} />
+    <meta property="og:image:height" content={`${image.height}`} />
+    <meta property="og:image:width" content={`${image.width}`} />
+    {/* twitter */}
+    <meta name="twitter:creator" content={`@${twitter}`} />
+    <meta name="twitter:site" content={`@${twitter}`} />
+    <meta name="twitter:title" content={title} />
+    <meta name="twitter:description" content={description} />
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta property="twitter:image" content={image.url}></meta>
```

Comment:
inlined it as there were some issues when you have a different components and use `Fragment`